Documentation/Products/UmbracoForms/Developer/Extending/Adding-a-Fieldtype.md

```diff
@@ -2,25 +2,34 @@
 
 *This builds on the "[adding a type to the provider model](Adding-a-Type.md)" chapter*
 
-Add a new class to the visual studio solution and make it inherit from Umbraco.Forms.Core.FieldType and override the Editor property.
+Add a new class to the Visual Studio solution with a using directive using `using Umbraco.Forms.Core;` and make it inherit from Umbraco.Forms.Core.FieldType with an override of the Editor property.
 
-In the empty constructor add the following information:
+An example CustomFormDataType new data type should contain the following information in it's constructor (remember to set the Id to be a unique GUID):
```

Comment:
This sentence is rather confusing, also note that the terminology is mismatched - Field Type vs Data Type. Maybe something like: "As an example, we'll create a new field type called 'CustomFormFieldType'. Add the following to the constructor (be sure to set the Id to be a unique GUID):